Apply example_ynh (#37)

ericgaspar opened the pull request:
- *Fix #35*
- *Fix #32*
- *Fix error during package_check when `Validating that the app movim can/can't be accessed with its URL...` => `An error occured, check syslog for more information An error occured,`*

yalh76 committed:
Apply example_ynh (#37)

ericgaspar commented:
!testme

ericgaspar committed:
Merge branch 'master' into testing

alexAubin commented on scripts/install:
```diff
@@ -54,148 +57,143 @@ ynh_app_setting_set --app=$app --key=admin --value=$admin
 # Download, check integrity, uncompress and patch the source from app.src
 ynh_setup_source --dest_dir="$final_path"
 
+ynh_replace_string --match_string="0.3.5" --replace_string="0.4.1" --target_file="$final_path/composer.json"
```
Ideally a comment wouldnt hurt to explain what this is about :stuck_out_tongue:

yalh76 replied:
Information added at https://github.com/YunoHost-Apps/movim_ynh/pull/38/commits/fb22b44a57b18d62a47cbbd7a47c96bc1fb2a2bb

alexAubin commented on scripts/upgrade:
```diff
@@ -56,47 +81,26 @@ if ynh_legacy_permissions_exists; then
 	# Download, check integrity, uncompress and patch the source from app.src
 	ynh_setup_source --dest_dir="$final_path"
+
+	ynh_replace_string --match_string="0.3.5" --replace_string="0.4.1" --target_file="$final_path/composer.json"
```
(same here)

yalh76 replied:
(same here)

yalh76 committed:
Adding more informations

to fix `movim.ERROR: Error: Call to undefined function GuzzleHttp\Psr7\uri_for() in /var/www/movim/vendor/ratchet/pawl/src/Connector.php (line 126) Trace #0 /var/www/movim/vendor/ratchet/pawl/src/Connector.php(38): Ratchet\Client\Connector->generateRequest('ws://127.0.0.1:...', Array, Array) #1 /var/www/movim/linker.php(323): Ratchet\Client\Connector->__invoke('ws://127.0.0.1:...', Array, Array) #2 {main} [] []`
